server/app/views/api/v1/api_users/create.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+
+
+#json.partial! 'profile', profile: @current_user
```

Comment:
```suggestion
```

Also: the empty lines at the top of the file can be removed.

Reply:
agree and updated